docs/modules/operation/pages/quick-start/inventory.adoc

```diff
@@ -0,0 +1,113 @@
+
+Organizing nodes with a similar network monitoring profile into a requisition (for example, routers or switches), lets you assign the same services, detectors, and policies for network monitoring behavior.
+
+A requisition includes a default foreign source definition that serves as a template to define the services to detect on (DNS, FTP, ICMP, and so on), the scan interval for discovery, and the policies to use when provisioning.
+
+While you can perform discovery without a requisition and edit it later, we recommend that you create a requisition before discovery.
+You can iteratively build a requisition and later actually import the requisition's nodes into {page-component-title} to process all add, changes, and deletes at once.
+
+This procedure describes how to create an empty requisition using the default foreign source definition.
+See the links at the end of the procedure for additional information on customizing a requisition.
+
+To create a requisition, follow these steps:
+
+. In the web UI, click the gear icon in the top right.
+. In the *Provisioning* area, choose *Manage Provisioning Requisitions*.
+. Click *Add Requisition*, type a name, and click *OK*.
+
+NOTE: The requisition remains red until you synchronize it with the database.
+Synchronize after you configure discovery.
+
+image::provisioning/red_requisition.png[]
+
+Once created, you can add nodes to the requisition.
+
+IMPORTANT: The requisition editor in the OpenNMS web UI is designed for small-scale use.
+Do not attempt to use this interface to edit requisitions containing more than a few dozen nodes.
+It is not designed for that use case and will fail.
+
+Additional configuration options for requisitions:
+
+* xref:deep-dive/provisioning/directed-discovery.adoc#directed-discovery[Manually specify nodes to add to a requisition]
+* xref:deep-dive/provisioning/auto-discovery.adoc#auto-discovery[Automatically discover nodes to add to a requisition]
+* Customize a requisition with xref:reference:provisioning/detectors.adoc#ref-detectors[detectors] and xref:deep-dive/provisioning/policies.adoc#policies[policies]
+
+[[configure-discovery]]
+== Configure discovery
+For this quick start, we assume that you do not have a list of nodes to start from.
+The procedure uses the default general settings for parameters like timeouts and retries.
+You can customize these for your own needs.
+
+To configure discovery, follow these steps:
+
+. In the web UI, click the gear icon in the top right.
+. In the *Provisioning* area, choose *Configure Discovery*.
+. In the *General Settings* area, select the requisition you just created from the *Requisition* list.
+.. (optional) Change the default values, if desired.
+. Click *Save and Restart Discovery*.
+. Go to *Manage Provisioning Requsitions* and click the *Synchronize the Requisition* icon, choose a scan option, and click *Synchronize*.
```

Comment:
Are these steps up to date? I couldn't find the Save and Restart Discovery button.

Reply:
Yes, it's in the top-left. Good user testing, though, so I'll add that in, since it's obviously not obvious. :-)